packages/code-studio/src/settings/FormattingSectionContent.test.tsx

```diff
@@ -111,7 +115,10 @@ describe('default decimal formatting', () => {
       }),
     });
 
-    userEvent.click(container.querySelector('.btn-reset-decimal'));
+    const element = container.querySelector('.btn-reset-decimal');
```

Comment:
Add a `test-id` to the button and then just use `screen.getByTestId('btn-reset-decimal')` - that's preferred to doing this.